bootstrap.sh

```diff
@@ -63,8 +63,7 @@ fi
 # everything under `external/`. That's why include both in the PYTHONPATH
 
 export PATH=$(pwd)/external/usr/bin:$PATH
-export PYTHONPATH=$(pwd)/external:$(pwd)/external/${libpath}/python$pyver/site-packages:$PYTHONPATH
-echo "Setting PYTHONPATH to ${PYTHONPATH}"
+export PYTHONPATH=$(pwd)/external:$(pwd)/external${libpath}/python$pyver/site-packages:$PYTHONPATH
```

Comment:
I think we should also a small check that the `libpath` is an absolute path and issue an error if not, or prefix it with `pwd`.